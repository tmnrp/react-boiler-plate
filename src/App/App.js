import React from "react";

const App = () => {
  return <div data-testid="myid">Hello World</div>;
};

export default App;
