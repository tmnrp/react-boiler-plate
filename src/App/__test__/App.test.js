import React from "react";
import "@testing-library/jest-dom/extend-expect";
import { render } from "@testing-library/react";
import App from "../App";

describe("App.js", () => {
  test("Should render properly", () => {
    const { getByTestId } = render(<App />);
    expect(getByTestId("myid")).toMatchSnapshot();
  });
});
