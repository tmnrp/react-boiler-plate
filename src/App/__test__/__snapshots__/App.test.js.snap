// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.js Should render properly 1`] = `
<div
  data-testid="myid"
>
  Hello World
</div>
`;
